logdna/client_test.go

```diff
@@ -0,0 +1,109 @@
+package logdna
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClient_View(t *testing.T) {
+	body := make(map[string](interface{}))
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		json.NewDecoder(r.Body).Decode(&body)
```

Comment:
it doesn't look like you're using the body anywhere, you can remove all of that and the `Decode()`s